app/views/handlers/list_add_question.py

```diff
@@ -41,6 +42,12 @@ def handle_post(self):
         self._list_item_id = self.questionnaire_store_updater.add_list_item(
             self.parent_block["for_list"]
         )
+        completed_location = Location(
+            section_id=self.current_location.section_id,
+            block_id=self.current_location.block_id,
+            list_item_id=self._list_item_id,
+            list_name=self.parent_block["for_list"])
+        self.questionnaire_store_updater.add_completed_location(completed_location)
```

Comment:
This is the add block, we don't need to track the add block's progress?

Reply:
See [other comment](https://github.com/ONSdigital/eq-questionnaire-runner/pull/1105#discussion_r1197924625) for reason why this is done

Reply:
In addition to the above reason, it also aids the debug dump as list item completeness shows by default

Reply:
Addressed in another comment